services/migration_service/migration_files/20210202145952_add_runs_idx_ts_epoch_flow_id.sql

```diff
@@ -0,0 +1,60 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+-- Others
+
+CREATE INDEX runs_v3_idx_ts_epoch ON runs_v3 (ts_epoch);
+
+CREATE INDEX runs_v3_idx_gin_tags ON runs_v3 USING gin (tags, system_tags);
+CREATE INDEX runs_v3_idx_gin_tags_combined ON runs_v3 USING gin ((tags || system_tags));
+
+-- flow_id + ts_epoch
+
+CREATE INDEX runs_v3_idx_flow_id_asc_ts_epoch_desc ON runs_v3 (flow_id ASC, ts_epoch DESC);
+CREATE INDEX runs_v3_idx_flow_id_desc_ts_epoch_desc ON runs_v3 (flow_id DESC, ts_epoch DESC);
+
+-- user && ts_epoch
+
+CREATE INDEX runs_v3_idx_user_asc_ts_epoch_desc ON runs_v3 (
```

Comment:
can you remind me again why it's not possible to just create an index on the user_name column without parsing the tags?
If the user_name column isn't being properly populated then that should be fixed

Reply:
from what I recall this was primarily implemented for the query for grouping by user. When a flow is run on SFN, the user_name is set to 'SFN', which slams that result set in between actual users, instead of what is preferred (NULLS LAST). @msavela have any addional info?